docs/api-docs/storefront/widgets/widget-versioning-tutorial.md

```diff
@@ -0,0 +1,246 @@
+
+To place your widget, send a `POST` request to [`/v3/content/placements`](https://developer.bigcommerce.com/api-reference/store-management/widgets/placement/createplacement).
+
+```http
+POST /stores/{{STORE_HASH}}/v3/content/placements
+Host: api.bigcommerce.com
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+{
+  "widget_uuid": "{your-widget-uuid}",
+  "entity_id": "",
+  "template_file": "pages/home",
+  "status": "active",
+  "sort_order": 1,
+  "region": "home_below_featured_products"
+}
+```
+
+[![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/store-management/widgets/placement/createplacement#requestrunner)
+
+The widget should now be visible on your store's homepage under **Featured Products**.
+
+## Update the widget template
+
+To create a new version of the widget template without affecting the widget created from it, we are going to include the `"create_new_version": true` parameter in the `PUT` request to [`/v3/content/widget-templates/{uuid}`](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget-template/updatewidgettemplate). This will create a new version record and associate that record to the current version. 
+
+```http
+POST /stores/{{store_hash}}v3/content/widget-templates/{uuid}
+Host: api.bigcommerce.com
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+{
+  "name": "Red Text",
+  "schema": [
+    {
+      "type": "tab",
+      "label": "Content",
+      "sections": [
+        {
+          "settings": [
+            {
+              "type": "text",
+              "label": "Text Content",
+              "id": "textContent",
+              "typeMeta": {
+                "maxLength": 1000
+              }
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "template": "<h1 style='color:red;'>Hello, World!</h1>",
+  "create_new_version": true
+}
+```
+
+[![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget-template/updatewidgettemplate#requestrunner)
+
+Let's look at our widget again. Send a `GET` request to [`/v3/content/widgets/{uuid}`](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget/getwidget) to retrieve your widget's data.
+You will notice that `current_version_uuid` has changed and is now different from the widget's `version_uuid`. Although, we have updated the widget template, the widget created using that template did not change.
+
+## Create a new widget
+
+All widgets created after the update, will be based on the most current version. Let's create another widget to test this functionality. 
```

Comment:
All widgets created after the update will be based on the most current version. Let's create another widget to test this functionality.